controlplane/pkg/apis/crossconnect/crossconnect.proto

```diff
@@ -14,22 +14,31 @@ enum CrossConnectEventType {
 
 message CrossConnectEvent {
     CrossConnectEventType type = 1;
-    map<string,CrossConnect> cross_connects = 2;
+    map<string, CrossConnect> cross_connects = 2;
+}
+
+enum CrossConnectState {
```

Comment:
I haven't fully wrapped my head around your PR yet, so you may have already done something exceptionally clever here... but if we put the state on the CrossConnect rather than the connections, how does the source nsmd find out that the dst nsmd has marked the remote connection as DOWN?

Reply:
Actually nothing clever I haven't done. I just subscribe src-nsmd to cross-connect monitor of dst-nsmd. I thought that we can leave only cross-connect monitor and eliminate connection monitor for simplicity reason. But redoing doesn't take long